ui/mage/fire/presets.ts

```diff
@@ -41,6 +41,44 @@ export const P1_EP_PRESET = PresetUtils.makePresetEpWeights(
 	}),
 );
 
+export const FIRE_BREAKPOINTS = new Map([
+	[
+		Stat.StatSpellHaste,
+		// Picked from Mage Discord
+		// Sources:
+		// https://docs.google.com/spreadsheets/d/17cJJUReg2uz-XxBB3oDWb1kCncdH_-X96mSb0HAu4Ko/edit?gid=0#gid=0
+		// https://docs.google.com/spreadsheets/d/1WLOZ1YevGPw_WZs0JhGzVVy906W5y0i9UqHa3ejyBkE/htmlview?gid=19
+		new Map([
+			['5-tick LvB/Pyro', 1602],
+			['12-tick Combust', 1922],
+			['13-tick Combust', 3212],
+			['14-tick Combust', 4488],
+			['6-tick LvB/Pyro', 4805],
+			['15-tick Combust', 5767],
+			['16-tick Combust', 7033],
+			['7-tick LvB/Pyro', 8000],
+			['17-tick Combust', 8309],
+			['18-tick Combust', 9602],
+			['19-tick Combust', 10887],
+			['8-tick LvB/Pyro', 11198],
+			['20-tick Combust', 12182],
+			['21-tick Combust', 13463],
+			['9-tick LvB/Pyro', 14412],
+			['22-tick Combust', 14704],
```

Comment:
Looks good! Mostly aligns with the breakpoints we setup for the softcaps so should yield nice results for manual optimization.

Reply:
Yea, I will expand the breakpoints config for soft capping once P2 gets announced